apps/docs/src/guide/errors/debugging-revert-errors.md

```diff
@@ -0,0 +1,39 @@
+# Debugging Revert Errors
+
+Unfortunately, the SDK does not support debugging revert errors with string messages - at least temporarily. This is because the SDK does not support decoding of Sway `str` slices in the `v0` Sway encoding scheme. This problem will be fixed soon once the `v1` Sway encoding scheme is adopted.
```

Comment:
Here, we state that the problem will be fixed in v1.

Can we have an issue for this so it doesn't go unnoticed?

We must be on top of this to guarantee this is indeed only temporary.

The new encoding is taking good shape already; this could be something to go top of it. cc @danielbate

Reply:
This has now been proved as working in #1832 and I've added a test for the revert failing gracefully, rather than unsupported str s;oce. I am only testing for `RequireFailed`, I will leave the specifics extracted from the logs to this PR.